scripts/backend_tests.py

```diff
@@ -24,15 +24,51 @@
 # pylint: disable=wrong-import-order
 import argparse
 import datetime
+import importlib
+import inspect
 import os
 import re
 import subprocess
+import sys
 import threading
 import time
 
 # pylint: enable=wrong-import-order
 
 
+CURR_DIR = os.path.abspath(os.getcwd())
+OPPIA_TOOLS_DIR = os.path.join(CURR_DIR, '..', 'oppia_tools')
+THIRD_PARTY_DIR = os.path.join(CURR_DIR, 'third_party')
+DIRS_TO_ADD_TO_SYS_PATH = [
+    os.path.join(OPPIA_TOOLS_DIR, 'coverage-4.5.3', 'coverage', 'cmdline'),
```

Comment:
It looks like this directory cannot be found. I wonder if it is actually being downloaded and installed correctly? Does the backend test script run the setup/install-third-party scripts first?

Reply:
Maybe you could try doing a listdir to the console to see what's in the folder -- see whether coverage.cmdline exists at all and maybe cat its contents to the terminal.